launch/utils.py

```diff
@@ -7,3 +7,45 @@ def trim_kwargs(kwargs_dict: Dict[Any, Any]):
     """
     dict_copy = {k: v for k, v in kwargs_dict.items() if v is not None}
     return dict_copy
+
+
+def infer_env_params(env_selector: Optional[str]):
+    """
+    Returns an env_params dict from the env_selector.
+
+    env_selector: str - Either "pytorch" or "tensorflow"
+    """
+    if env_selector == "pytorch":
+        import torch
+
+        try:
+            ver = torch.__version__.split("+")
+            torch_version = ver[0]
+            cuda_version = ver[1][2:] if len(ver) > 1 else "113"
+            if (
+                len(cuda_version) < 3
+            ):  # we can only parse cuda versions in the double digits
+                raise ValueError(
+                    "PyTorch version parsing does not support CUDA versions below 10.0"
+                )
+            tag = f"{torch_version}-cuda{cuda_version[:2]}.{cuda_version[2:]}-cudnn8-runtime"
```

Comment:
do we also need to check for python versions at some point? (to be fully correct at least)
since the pytorch images seem bundled with different python versions also (oh boy)

Reply:
just added python versions as u commented LOL